index.js

```diff
@@ -30,8 +31,16 @@ module.exports = (options, callback) => {
     }, (err, syms) => {
       if (err) callback(err)
       const concatted = syms.reduce((m, o) => m.concat(o), [])
-      const sorted = concatted.sort((a, b) => a.i - b.i)
-      callback(null, sorted.map(x => x.symbol))
+
+      let split = content.split('\n')
+      concatted.map((sym) => {
+        split[sym.i] = sym.symbol
+      })
+      const parsed = path.parse(file)
+      const dest = path.join(parsed.dir, parsed.name + '_symbolicated' + parsed.ext)
+      fs.writeFileSync(dest, split.join('\n'))
```

Comment:
I very often use `atos` with `pbpaste` like:

```
$ pbpaste | npx @electron/symbolicate-mac --file /dev/stdin --version 6.0.0-beta.15
```

looks like this breaks existing usage by outputting to a file instead of stdout?

Reply:
Yeah, outputtting to a file is not necessary. It's easy to output `stdio` to file if needed. I'll change the code.